client/src/components/cards/CardsIndex.jsx

```diff
@@ -16,13 +30,29 @@ class CardsIndex extends Component {
   }
 
   render() {
+    const { classes } = this.props;
```

Comment:
whats the reason for the { }

Reply:
{} is destructuring props to access the classes passed by withStyles higher order component.